cli/src/pcluster/config/cluster_config.py

```diff
@@ -371,17 +372,27 @@ def __init__(self, mount_dir: str, name: str):
     def _register_validators(self, context: ValidatorContext = None):  # noqa: D102 #pylint: disable=unused-argument
         self._register_validator(SharedStorageNameValidator, name=self.name)
 
+    @property
+    def unmanaged(self):
```

Comment:
TIP - tipically we use "is_" prefix to describe that a boolean value is returned.

Reply:
Fixed